TaskParallelismFramework/springApi/pom.xml

```diff
@@ -42,7 +42,7 @@
 		</dependency>
 		<dependency>
 			<groupId>sjsu.cmpe.B295</groupId>
-			<artifactId>TaskParallelism</artifactId>
+			<artifactId>TaskParallelization</artifactId>
```

Comment:
@Vaishnavi109 @regoclint Please check if this is correct?
@darshanmodh : any specific reason for this?